tests/pytests/unit/modules/test_win_lgpo_reg.py

```diff
@@ -0,0 +1,252 @@
+import pytest
+
+import salt.modules.win_lgpo_reg as lgpo_reg
+import salt.utils.files
+import salt.utils.win_lgpo_reg
+from salt.exceptions import SaltInvocationError
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+    pytest.mark.skip_unless_on_windows,
+    pytest.mark.destructive_test,
+]
+
+
+@pytest.fixture(scope="function")
```

Comment:
```suggestion
@pytest.fixture
```
Fixtures are function scoped by default

Reply:
Done